src/Mod/Arch/ArchWindow.py

```diff
@@ -531,6 +531,14 @@ def __init__(self,obj):
         obj.IfcType = "Window"
         obj.MoveWithHost = True
 
+        try:
+            import ArchSketchObject
+            print(" Done :  __init__() import ArchSketchObject ")
+            ArchSketchObject.ArchSketch.setPropertiesLinkCommon(self, obj)  # ,linkFp)
+            print(" Done :  __init__() ArchSketchObject.setPropertiesLinkCommon() ")
+        except:
+            print(" ArchSketchObject add-on module is not installed yet, __init__() ")
+
```

Comment:
I would do all these blocks like this, simply:

```suggestion
      try:
            # use the ArchSketchObject addon if available
            import ArchSketchObject
            ArchSketchObject.ArchSketch.setPropertiesLinkCommon(self, obj)  # ,linkFp)
        except:
            pass
```

Reply:
Thanks reviewing :)

I had revamped early in Sept, maybe I try ArchEquipment first.

Reply:
#5241 Still something wrong with git ... :(

Reply:
Completely start over cloning everything again and open a new PR #5243, closed the #5241.

Hope do it right this time :)